test/development/basic/styled-components/next.config.js

```diff
@@ -1,5 +1,8 @@
 module.exports = {
   compiler: {
-    styledComponents: true,
+    styledComponents: {
+      displayName: true,
```

Comment:
Should we leave this unset to ensure it is automatically enabled in development? 

Also is there an assertion we can add for this in https://github.com/vercel/next.js/blob/canary/test/development/basic/styled-components.test.ts?

Reply:
Can do, this was kinda left over from testing 👍

Reply:
done